doc: update 1.6 upgrade guide (#1592)
apache/apisix-ingress-controller: docs/en/latest/upgrade.md

#
-->

## Version change

### ***1.6.0***

> The APISIX 3.x.x has adjusted the admin-api, in order to make APISIX Ingress compatible with these changes, You need to select the corresponding chart version, please refer [1.5 to 1.6](#15-to-16).

Reviewer: ```suggestion
The APISIX 3.x.x has changed the admin-api, in order to make APISIX Ingress compatible with these changes, You need to select the corresponding chart version, please refer [1.5 to 1.6](#15-to-16).
```

> The APISIX 3.x.x has adjusted the admin-api, in order to make APISIX Ingress compatible with these changes, You need to select the corresponding chart version, please refer [1.5 to 1.6](#15-to-16).

Reviewer 2: ```suggestion
APISIX 3.x.x has changed the Admin API. To make APISIX Ingress compatible with these changes, you need to select the corresponding chart version. Please refer [1.5 to 1.6](#15-to-16).
```

### ***1.5.0***

- CRD has been upgraded to the V2 version, and V2beta3 has been marked as deprecated.

> - `app_namespace` is deprecated, you can use `namespace_selector` instead.

Reviewer 2: ```suggestion
- `app_namespace` is deprecated,. You can use `namespace_selector` instead.
```

  --set ingress-controller.config.apisix.serviceNamespace=ingress-apisix
```

### Compatible upgrade

Compatible upgrades can be made without changing any resources.

#### ***1.5 to 1.6***

[Relevant version information and compatibility of apisix-helm-chart](https://github.com/apache/apisix-helm-chart#compatibility-matrix).

> You need to select the corresponding chart version according to the APISIX version to install or upgrade. As shown in the following table:

Reviewer 2: ```suggestion
You need to select the corresponding chart version according to the APISIX version as shown to install or upgrade:
```
